services/common/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
   implementation 'org.springframework:spring-web:' + '5.3.18'
   implementation 'org.springframework:spring-webmvc:' + '5.3.18'
   implementation 'org.springframework.boot:spring-boot-starter-webflux:2.7.8'
+  implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.7.8'
```

Comment:
Do we need to add this explicitly or it was unintentionally added?

Reply:
Explicitly added for pages.